SetupNode/scripts/prepare_remote_machine.py

```diff
@@ -160,29 +185,35 @@ def format_labels_command():
     if 'MANAGEMENT' in os.environ:
         res += ' --label-add management=yes'
 
-    return 'docker node update {0} {1}'.format(res, os.environ['REMOTE_HOST_NAME'])
+    return 'docker node update {0} {1}'.format(res, hostname)
 
 def run_consul_reshafle_command():
```

Comment:
We don't need to reshuffle since the recommendation is to define consul as global